addons/account/views/account_move_views.xml

```diff
@@ -774,7 +774,7 @@
                         <div class="oe_title">
                             <!-- Invoice draft header -->
                             <span class="o_form_label"><field name="move_type" attrs="{'invisible': [('move_type', '=', 'entry')]}" readonly="1" nolabel="1"/></span>
-                            <h1 attrs="{'invisible': ['|', ('state', '!=', 'draft'), ('name', '!=', '/')]}">
+                            <h1 attrs="{'invisible': ['|', '|', ('state', '!=', 'draft'), ('name', '!=', '/'), ('posted_before', '=', True)]}">
```

Comment:
Do we still need the condition on the name ? Shouldn't the one on posted before replace it ? :thinking:

Reply:
It is possible to have a Draft Invoice that has a name that is not '/'. Create a new journal (or new company). The first invoice will have a name / sequence number (so that it can be adapted if it is not right). If we remove the condition it would say "Draft" and the name / number. I checked that this should not happen.